src/applications/claims-status/tests/e2e/page-objects/TrackClaimsPageV2.js

```diff
@@ -0,0 +1,455 @@
+    cy.url().should('contain', 'ask-va-to-decide');
+    cy.get('.main .button-secondary').click();
+    cy.url().should('contain', 'status');
+    cy.axeCheck();
+  }
+
+  submitFilesForReview() {
+    cy.intercept('POST', `/v0/evss_claims/189685/documents`, {
+      body: {},
+    }).as('documents');
+    cy.get('.usa-file-input input')
+      .selectFile({
+        contents: Cypress.Buffer.from('test file contents'),
+        fileName: 'file-upload-test.txt',
+        mimeType: 'text/plain',
+        lastModified: Date.now(),
+      })
+      .then(() => {
+        cy.get('.document-item-container va-select')
+          .shadow()
+          .find('select')
+          .select('L029');
+      });
+    cy.get('.additional-evidence-container va-checkbox')
+      .shadow()
+      .find('input[type="checkbox"]')
+      .check({ force: true });
+    cy.get('va-button#submit')
+      .shadow()
+      .find('button')
+      .click();
+    cy.wait('@documents');
+    cy.get('va-alert h2').should('contain', 'We have your evidence');
+  }
+
+  submitFilesShowsError() {
+    cy.get('va-button#submit')
+      .shadow()
+      .find('button')
+      .click()
+      .then(() => {
+        cy.get('va-file-input')
+          .shadow()
+          .find('.usa-error-message')
+          .should('contain', 'Please select a file first');
+        cy.injectAxeThenAxeCheck();
+      });
+  }
+
+  verifyContentions() {
+    cy.get('.claim-contentions > ul > li').should('have.length', 3);
+    cy.get('.claim-contentions > ul > li:nth-child(4)').should('not.exist');
+    cy.get('.show-all-button').click();
+    cy.get('.claim-contentions > ul > li').should('have.length', 4);
+  }
+
+  verifyRecentActivity() {
+    cy.get('.recent-activity-container').should('be.visible');
+    cy.get('.recent-activity-container > h3').should(
+      'contain',
+      'Recent activity',
+    );
+    cy.get('.recent-activity-container > ol > li').should(
+      'have.length.greaterThan',
+      0,
+    );
+  }
+
+  verifyRecentActivityPagination() {
+    cy.get('.recent-activity-container').should('be.visible');
+    cy.get('.recent-activity-container > h3').should(
+      'contain',
+      'Recent activity',
+    );
+    cy.get('.recent-activity-container > ol > li').should('have.length', 10);
+    cy.get('va-pagination').should('be.visible');
+    cy.get('va-pagination')
+      .shadow()
+      .get('.usa-pagination__list > li')
+      .should('have.length.greaterThan', 1);
+  }
+
+  verifyPrimaryAlert() {
+    cy.get('va-alert.primary-alert')
+      .first()
+      .should('be.visible');
+    cy.get('va-alert.primary-alert')
+      .first()
+      .shadow()
+      .get('va-alert.primary-alert:first-of-type a')
+      .should('contain', 'View details');
+    cy.get('va-alert.primary-alert')
+      .first()
+      .shadow()
+      .get('va-alert.primary-alert:first-of-type a')
+      .click();
+    cy.url().should('not.contain', '/your-claims/189685/status');
```

Comment:
maybe this should instead confirm the new link `http://localhost:3001/track-claims/your-claims/189685/document-request/5`